src/zeek-setup.cc

```diff
@@ -480,8 +480,8 @@ static std::vector<std::string> get_script_signature_files()
 	std::vector<std::string> rval;
 
 	// Parse rule files defined on the script level.
-	char* script_signature_files = util::copy_string(
-		id::find_val("signature_files")->AsString()->CheckString());
```

Comment:
That's not doing the same thing.

`strsep(&tmp, " \t")` will split the string at *either* whitespace or tab. If I call `util::split(s, " \t")`, it will split the string at `" \t"` instead (not either one, but both characters).

Reply:
Grumble - I missed the `" "` there... the following and keep the `strsep()` munging as is?

```
auto sig_files = id::find_val("signature_files")->AsStringVal()->ToStdString();
char *s = sig_files.data();
```

Reply:
Done.